packages/blade/src/components/Chip/__tests__/Chip.web.test.tsx

```diff
@@ -39,22 +39,6 @@ describe('<Chip />', () => {
     expect(getByRole('radio', { name: 'Mango' })).toBeInTheDocument();
   });
 
-  it('should render chip with intent', () => {
```

Comment:
this shall ideally fail right? instead of removing this test maybe check for failure?

Reply:
TS will throw an error. The component won't throw any runtime error because we don't have any runtime checks. We have removed intent/variant tests in other components too.